src/Symfony/Component/Messenger/Transport/Doctrine/Connection.php

```diff
@@ -36,6 +36,7 @@ class Connection
         'queue_name' => 'default',
         'redeliver_timeout' => 3600,
         'auto_setup' => true,
+        'redelivery_max_count' => 3,
```

Comment:
I don't think this should be configurable as it's an implementation detail people should not care about. It's also not documented currently. Just making it a private const on the DoctrineReceiver is enough.

Reply:
I also updated the documentation symfony/symfony-docs#12109

but... I will be glad, if someone else comment it if yes or not.

Reply:
@nicolas-grekas / @sroze any suggestions?

Reply:
Ok... I removed this part of configuration...